src/pages/workspace/AccessOrNotFoundWrapper.tsx

```diff
@@ -160,19 +163,9 @@ function AccessOrNotFoundWrapper({accessVariants = [], fullPageNotFoundViewProps
         );
     }
 
-    return callOrReturn(props.children, props);
+    return callOrReturn(props.children, {report, policy, isLoadingReportData});
```

Comment:
Verify that this doesn't break anything (there are less props provided to children now)

Reply:
We only use it here, so maybe let's narrow the type down to include `policy` only?

https://github.com/Expensify/App/blob/bb30c66795d8f4c5c713ee09d90946e2ed65cda8/src/pages/workspace/tags/WorkspaceTagsSettingsPage.tsx#L125